app/views/provider/admin/destroys/index.html.erb

```diff
@@ -1,25 +0,0 @@
-<% content_for :page_header_title, "Deleted #{@kind.human.pluralize}" %>
```

Comment:
I wasn't aware of this feature, to be honest. But now when I see it, maybe it's actually useful :P We can as @3scale/support if they use it.

But well, if we decide to delete it, I'd say we also need to cleanup routes and the controller.

Reply:
Yeah you're right it probably deserves it's own PR